src/include/sof/drivers/acp_dai_dma.h

```diff
@@ -0,0 +1,44 @@
+/* SPDX-License-Identifier: BSD-3-Clause
+ *
+ * Copyright(c) 2021 AMD. All rights reserved.
+ *
+ * Author:	Basavaraj Hiregoudar <basavaraj.hiregoudar@amd.com>
+ *		Anup Kulkarni<anup.kulkarni@amd.com>
+ *		Bala Kishore <balakishore.pati@amd.com>
+ */
+#ifndef __SOF_DRIVERS_EDMA_H__
+#define __SOF_DRIVERS_EDMA_H__
+
+#include <sof/bit.h>
+#include <sof/trace/trace.h>
+#include <user/trace.h>
+
+#if CONFIG_AMD_BT
+#define ACP_BT_ENABLE
+#endif
+
+int acp_dma_init(struct sof *sof);
+
+#define DMA_BUFFER_PERIOD_COUNT	2
+
+/* default max number of channels supported */
+#define DEFAULT_NUM_CHANNELS    2
+
+/* default sample rate */
+#define DEFAULT_SAMPLE_RATE     48000
+
+#define ACP_DMA_BUFFER_ALIGN	64
+#define ACP_DMA_TRANS_SIZE	64
+#define AMD_DAI_DMA_BUFFER_PERIOD_COUNT		2
+#define DRAM_ADDRESS_MASK			0x0FFFFFFF
```

Comment:
Some of the macros also need a AMD_ or ACP_ prefix as naming is generic and may conflict.

Reply:
Addressed the Review Comment. Done